### PR TITLE
Update HIRING.md

### PR DESCRIPTION
**Pull request by elland**
Changed the language to be more inclusive, and the requirements to fit more with what we do. :)

### CODE REVIEW

File: HIRING.md [hunk 1]
```diff
@@ -47,27 +47,26 @@ We’re obviously trying to assess the candidate but remember they are assessing
...
-- 03 A broad understanding of Swift, Core Data and Web-services.
-- 04 Self-sustaining and forever curious about how to improve your workflow.
-- 05 Experience with unit testing.
-- 06 Further user experience knowledge is a bonus.
-- 07 or collaborating on open source projects is a big plus.
+- 01 Familiarity with building applications for iOS and/or Mac OS. Maybe even some Watch and tvOS.
+- 02 Teamplayer.
+- 03 Familiarity or interest in Swift, usability and Web services.
+- 04 Self-sustaining and forever curious.
+- 05 Familiarity with unit testing.
```
Comment (3lvis) [inline HIRING.md:59]: What's the reasoning of changing Experience with Familiarity?

Comment (elland) [reply HIRING.md:59]: Asking for experience is exclusionary, and we're not necessarily looking for a senior developer, but we're also open for juniors.

Comment (3lvis) [reply HIRING.md:59]: Good point 👍

File: HIRING.md [hunk 2]
```diff
@@ -47,27 +47,26 @@ We’re obviously trying to assess the candidate but remember they are assessing
...
 
 ### A bit about you
-- 01 Experience with building top level applications for Apple iOS/OS X, and started peeping into Watch OS and tvOS.
-- 02 Extensive experience with team and client work.
-- 03 A broad understanding of Swift, Core Data and Web-services.
-- 04 Self-sustaining and forever curious about how to improve your workflow.
-- 05 Experience with unit testing.
-- 06 Further user experience knowledge is a bonus.
-- 07 or collaborating on open source projects is a big plus.
+- 01 Familiarity with building applications for iOS and/or Mac OS. Maybe even some Watch and tvOS.
```
Comment (marijnschilling) [inline HIRING.md]: I think just iOS is enough. When have me ever made an OS X app?

Comment (elland) [reply HIRING.md]: I wouldn't rule out hiring a Mac developer to work on iOS.

Comment (marijnschilling) [reply HIRING.md]: Me neither! But let's not put it in the job requirements that prominently either

Comment (3lvis) [reply HIRING.md]: @Elland Mac developer with Familiarity on iOS fits in the simpler description.

Comment (elland) [reply HIRING.md]: 🤔 what would you suggest?

Comment (3lvis) [reply HIRING.md]: `Familiarity with building applications for iOS.`

Comment (elland) [reply HIRING.md]: Fine by me. 🆗

Comment (marijnschilling) [reply HIRING.md]: Or maybe:
`You enjoy building applications for iOS`
Or something in that tone? The repetition of the word "Familiarity" (or Experience for that matter) annoys me a bit 😅

Comment (elland) [reply HIRING.md]: Repetition is indeed a bit annoying, but I don't think enjoyment should be a requirement. 😂

Comment (3lvis) [reply HIRING.md]: Let's merge it first then tweak it.

**Pull request merged**